cogs/economy.py

```diff
@@ -220,8 +220,11 @@ def __init__(self, bot):
     async def register(self, ctx):
         """Registers an account at the Twentysix bank"""
         user = ctx.message.author
+        credits = 0
+        if ctx.message.server.id in self.settings:
+            credits = self.settings[ctx.message.server.id]["REGISTER_CREDITS"]
         try:
-            account = self.bank.create_account(user)
+            account = self.bank.create_account(user, credits)
```

Comment:
This line MUST be `account = self.bank.create_account(user, initial_balance=credits)` because of the damned `*` operator.